hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.hudi.util;
 
 import org.apache.hudi.common.model.HoodieRecordLocation;
+import org.apache.hudi.common.model.HoodieTableType;
```

Comment:
Do we meet a compile error?

Reply:
@yanghua indeed strange. CI should have failed for [this PR](https://github.com/apache/hudi/commit/bc883db5de5832fa429bbb04a35d3606fdacdb2a#diff-819d51e0aa6f24c56d99119ad4888e85db6ad51a33545ac5062ba86136fd0d92). Looks like Travis went blind for a while 😞

Reply:
Before you merge the code, at lease you need to see if the `HoodieTableType` is used in the master code.

Reply:
That's weird ~